client/app/page.tsx

```diff
@@ -22,80 +41,200 @@ export default function HomeScreen() {
+          <button
+            style={styles.item}
+            onClick={() => triggerKeyPress(serverIP, 'right_arrow')}
+          >
+            <Image
+              width='25'
+              height='25'
+              alt='right arrow key'
+              src={`/images/light/arrow-right.png`}
+            />
           </button>
         </div>
 
-      </div>
+        <div style={styles.buttonRow}>
+          <BrightnessControls serverIP={serverIP} />
+
+          <div style={styles.column}>
+            <div
+              className={poppins.className}
+              style={{ width: '60px', margin: '10px' }}
+            >
+              <AnimatedProgressProvider easingFunction={easeQuadInOut} repeat>
+                {() => {
+                  const roundedValue = Math.round(+volume);
+                  return (
+                    <CircularProgressbar
+                      strokeWidth={6}
+                      value={+volume}
+                      text={`${roundedValue}`}
+                      styles={buildStyles({
+                        // Rotation of path and trail, in number of turns (0-1)
+                        rotation: 0.5,
 
-      <div>volume: {volume}</div>
+                        // Whether to use rounded or flat corners on the ends - can use 'butt' or 'round'
+                        strokeLinecap: 'round',
+
+                        // Text size
+                        textSize: '24px',
+
+                        // How long animation takes to go from one percentage to another, in seconds
+                        pathTransition: 'none',
+
+                        // Can specify path transition in more detail, or remove it entirely
+                        // pathTransition: 'none',
+
+                        // Colors
+                        pathColor: `rgba(255, 255, 255)`,
+                        textColor: '#FFFFFF',
+                        trailColor: 'rgba(0, 0, 0, 0)',
+                        backgroundColor: '#3e98c7',
+                      })}
+                    />
+                  );
+                }}
+              </AnimatedProgressProvider>
+            </div>
+            <button
+              style={styles.item}
+              onClick={() => reScan(i, incrRescan, setServerIP)}
+            >
+              <Image
+                width='25'
+                height='25'
+                alt='rescan'
+                src={`/images/light/rescan.png`}
+              />
+            </button>
+          </div>
+
+          <VolumeControls setVolume={setVolume} serverIP={serverIP} />
+        </div>
+
+        <div style={styles.colorModalButton}>
+          <button style={styles.item} onClick={openModal}>
+            <Image
+              width='25'
+              height='25'
+              alt='colorpicker'
+              src={`/images/light/color-picker.png`}
+            />
+          </button>
+        </div>
+
+        <Modal
```

Comment:
same here, can you create a new component for this under /components to keep pages.tsx clean?